tests/appium/project/src/test/kotlin/br/com/zup/beagle/cucumber/steps/ScrollViewScreenSteps.kt

```diff
@@ -16,135 +16,355 @@
 import io.cucumber.java.en.Then
 import io.cucumber.java.en.When
+import org.junit.Assert
+import org.openqa.selenium.By
 
 private const val SCROLLVIEW_SCREEN_HEADER = "Beagle ScrollView"
-private const val PARAGRAPH =
-    """Lorem ipsum diam luctus mattis arcu accumsan, at curabitur hac in dictum senectus neque,
-    orci lorem aenean euismod leo. eu nunc tellus proin eget euismod lorem curabitur habitant nisi himenaeos 
-    habitasse at quam, convallis potenti scelerisque aenean habitant viverra mollis fusce convallis dui
-    urna aliquam. diam tristique etiam fermentum etiam nunc eget vel, ante nam eleifend habitant per senectus diam,
-    bibendum lectus enim ultrices litora viverra. lorem fusce leo hendrerit himenaeos elementum aliquet nec,
-    vestibulum luctus pretium diam tellus ligula conubia elit, a sodales torquent fusce massa euismod. et magna 
-    imperdiet conubia sed netus vitae justo maecenas proin lorem, sapien nisi porttitor dolor facilisis pharetra 
-    nam class. Morbi nullam odio accumsan quam urna sit tortor vulputate mi fames, elit molestie gravida ipsum
-    dictumst aenean curabitur ultrices consectetur pharetra, auctor aenean diam pellentesque condimentum risus 
-    diam scelerisque rutrum. conubia sem tincidunt cras venenatis tristique nisl duis rhoncus blandit, sed mattis 
-    vulputate accumsan suscipit tristique imperdiet dui, ornare ipsum tempor viverra elementum consectetur euismod
-    dapibus. ultricies in consectetur libero nam ultrices egestas quis volutpat ut nec sagittis eu, elementum
-    malesuada ullamcorper dapibus donec aenean mattis odio mi nulla gravida. tellus metus imperdiet justo mattis 
-    eros sodales potenti nibh nisl tincidunt, metus etiam cubilia amet donec primis sapien erat dictumst. Accumsan 
-    etiam himenaeos tempor integer habitasse curae ac, tincidunt laoreet taciti nisl habitasse conubia, maecenas nec
-    velit vitae amet varius. scelerisque vel fringilla consequat justo curabitur nam massa vitae, tempus tempor 
-    sit torquent massa malesuada ullamcorper, laoreet elementum nam pharetra tempus nam mauris. sociosqu dictum 
-    malesuada lectus suscipit ullamcorper aliquet pulvinar semper laoreet, vulputate aliquam nibh odio donec ligula
-     bibendum suspendisse, facilisis ut lobortis lacus tortor hendrerit integer posuere. phasellus egestas dui hac 
-    auctor faucibus purus accumsan arcu, sem vivamus rhoncus pharetra aliquam ornare curabitur rutrum, ut venenatis
-     proin iaculis orci gravida molestie."""
+private const val NEW_TEXT_PREFIX = "Lorem ipsum diam luctus"
+private const val NEW_TEXT_SUFFIX = "proin iaculis orci gravida molestie."
 
 class ScrollViewScreenSteps : AbstractStep() {
     override var bffRelativeUrlPath = "/scrollview"
 
+    // cache
+    companion object {
+        var scrollViewElement1: MobileElement? = null
+        var scrollViewElement2: MobileElement? = null
+        var scrollViewElement3: MobileElement? = null
```

Comment:
Instead of creating nullable variables and having to use `!!` operators every time you access them, you can use the the `lateinit` modifier, as they will always be set before reading.
```suggestion
        lateinit var scrollViewElement1: MobileElement
        lateinit var scrollViewElement2: MobileElement
        lateinit var scrollViewElement3: MobileElement
```